### ISSUE TITLE
4.6.11 image disabled ncSOS

### ISSUE THREAD

**Issue by ebridger** [Unidata/thredds-docker#184]
Just updated our thredds-docker from 4.6.10 to 4.6.11 and noticed that our ncSOS services were giving an error message that ncSOS was not enabled.  Checking [threddsConfig.xml](https://www.unidata.ucar.edu/software/thredds/v4.6/tds/reference/ThreddsConfigXMLFile.html#ncISO) I find no mention of ncSOS or the need to enable it.  Never had to in the past.

But the [annotated example](https://raw.githubusercontent.com/Unidata/thredds/master/tds/src/main/webapp/WEB-INF/altContent/startup/threddsConfig.xml) shows the need for:
```
<!--
  <NCSOS>
    <allow>false</allow>
  </NCSOS>
  -->
```
Un-commenting and changing this to in our threddsConfig.xml  `<allow>true</allow>`   fixed the problem.
The docker  [threddsConfig.xml](https://github.com/Unidata/thredds-docker/blob/master/files/threddsConfig.xml) appears to be outdated, does not mention NCSOS.

BTW our thredds-docker instance using ncSOS is at: http://www.neracoos.org/thredds/sos_catalog.html
Really great that THREDDS has been dockerized. Thanks.

**Comment by julienchastang** [comment]
Related to #148 and https://github.com/Unidata/thredds/issues/48. Indeed I think https://github.com/Unidata/thredds/pull/826 made it onto the `4.6.11` branch.  @lesserwhirls and @cwardgar may have additional comments.

**Comment by lesserwhirls** [comment]
Yes, a [change](https://github.com/asascience-open/ncSOS/pull/187) was made to ncSOS once we added it as part of the standard TDS war to make it like the most of the other services in TDS 4.6.x (disabled by default, and as @ebridger found, can be enabled in `threddsConfig.xml`). In the next major version of the TDS, it will be enabled by default. The `threddsConfig.xml` file in this repo needs to be updated with the addition of:

```
  <!--
  <NCSOS>
    <allow>false</allow>
  </NCSOS>
  -->
```

**Comment by lesserwhirls** [comment]
Fixed in pr Unidata/thredds-docker#185
